doc/intro.md

```diff
@@ -192,25 +192,15 @@ She picks 113 say, and what ends up on the blockchain is:
 
     Y - Xi = (113*G + 3*H) - (28*G + 3*H) = 85*G + 0*H
 
-Now the transaction no longer sums to zero and we have an _excess value_ on _G_
-(85), which is the result of the summation of all blinding factors. But because `85*G` is
-a valid public key on the elliptic curve _C_, with private key 85,
-for any x and y, only if `y = 0` is `x*G + y*H` a valid public key on the elliptic curve 
-using generator point _G_.
+Now the transaction no longer sums to zero and we have an _excess value_ (85), which is the result of the summation of all blinding factors. Because `85*G` is
+a valid public key for the generator point _G_ the input and output values must sum to zero and the transaction is thus valid, since `x*G + y*H` is a valid public key for generator point _G_ if and only if `y = 0`.
 
-So all the protocol needs to verify is that (`Y - Xi`) is a valid public key on the curve 
-and that the transacting parties collectively know the private key `x` (85 in our transaction with
-Carol) of this public key. If they can prove that they know the private key to `x*G + y*H` using
-generator point _G_ then this proves that `y` must be `0` (meaning above that the sum of all 
-inputs and outputs equals `0`).
+So all the protocol needs to verify is that (`Y - Xi`) is a valid public key for generator point _G_ 
+and that the transacting parties collectively can produce its private key (85 in the above example). The simplest way to do so is to require a signature built with the excess value (85),
+which then ensures that:
 
-The simplest way to do so is to require a signature built with the excess value (85),
-which then validates that:
-
-* The transacting parties collectively know the private key (the excess value 85), and
-* The sum of the transaction outputs, minus the inputs, sum to a zero value
-  (because only a valid public key, matching the private key, will check against
-  the signature).
```

Comment:
Seems like we are loosing a bit of explanation here? I'd be for repeating things again just to clarify as much as possible.

Reply:
I didn't just delete that part. I rephrased it on lines 202-203.

Reply:
👍